### PR TITLE
[CLI-2584] Remove `context create` and `context update`

### PR DESCRIPTION
**Pull request by sgagniere**
Release Notes
-------------

Breaking Changes
- `confluent context create` and `confluent context update` has been removed

New Features
- PLACEHOLDER

Bug Fixes
- PLACEHOLDER

Checklist
---------
- [x] Leave this box unchecked if features are not yet available in production

What
----
The ability to create and maintain custom contexts is obsolete now that `confluent kafka topic produce | consume` works without login for both on-prem and cloud.

References
----------

Test & Review
-------------

### CODE REVIEW

File: internal/context/command_update.go [hunk 1]
```diff
@@ -18,11 +15,10 @@ func (c *command) newUpdateCommand() *cobra.Command {
 		RunE:              c.update,
```
Comment (brianstrauch) [inline internal/context/command_update.go:15]: Wait we should also delete this command, right? I think we have nothing to update if you can't create a Kafka-based context in the first place.

Comment (sgagniere) [reply internal/context/command_update.go:15]: I thought about it, but it still has some functionality: you can set the current Kafka cluster for a context.

Should we remove that in favor of `confluent kafka cluster use`? The only difference is that you'll have to switch contexts first.

Comment (brianstrauch) [reply internal/context/command_update.go:15]: > Should we remove that in favor of confluent kafka cluster use?

Yep, I think so.

Comment (brianstrauch) [reply internal/context/command_update.go:15]: (as long as no functionality is lost, we'd prefer having one standardized way to do this)

File: pkg/errors/typed.go [hunk 2]
```diff
@@ -127,7 +127,7 @@ func NewCorruptedConfigError(format, contextName, configFile string) CLITypedErr
 	return &CorruptedConfigError{
 		errorMsg: fmt.Sprintf("corrupted CLI config: %v", errorWithStackTrace),
 		suggestionsMsg: fmt.Sprintf("Your configuration file \"%s\" is corrupted.\n"+
-			"Remove config file, and run `confluent login` or `confluent context create`.\n"+
+			"Remove config file, and run `confluent login`.\n"+
```
Comment (brianstrauch) [inline pkg/errors/typed.go:130]:
```suggestion
			"Remove it, and run `confluent login`.\n"+
```

**Review by brianstrauch — APPROVED** [review]

**Review by brianstrauch — CHANGES_REQUESTED** [review]
Sorry to backpedal here, but I think we still want to support `confluent context update --name <name>`. Not sure what I was thinking when I opened this ticket, but we still want to allow users to rename contexts because they're currently super verbose... and renaming them makes it easier to do `confluent context use <name>` later.

File: pkg/errors/typed.go [hunk 3]
```diff
@@ -127,7 +127,7 @@ func NewCorruptedConfigError(format, contextName, configFile string) CLITypedErr
 	return &CorruptedConfigError{
 		errorMsg: fmt.Sprintf("corrupted CLI config: %v", errorWithStackTrace),
 		suggestionsMsg: fmt.Sprintf("Your configuration file \"%s\" is corrupted.\n"+
-			"Remove config file, and run `confluent login` or `confluent context create`.\n"+
+			"Remove it, and run `confluent login`.\n"+
```
Comment (brianstrauch) [inline pkg/errors/typed.go:130]: Nit: More precise wording
```suggestion
			"Delete it, and run `confluent login`.\n"+
```

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**